jumanji/environments/routing/rware/viewer.py

```diff
@@ -0,0 +1,332 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# flake8: noqa: CCR001
+
+from typing import Any, Callable, Optional, Sequence, Tuple
+
+import chex
+import matplotlib.animation as animation
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.collections import LineCollection
+from numpy.typing import NDArray
+
+import jumanji
+from jumanji.environments.routing.rware.types import Direction, State
+from jumanji.tree_utils import tree_slice
+from jumanji.viewer import Viewer
+
+RAD2DEG = 57.29577951308232
```

Comment:
Wouldn't it be better to define these constants in a different constants.py file?

Reply:
You are right, remove it.

Reply:
I also moved the constants out to a separate file as suggested by @ElshadaiK